packages/core-components/src/components/CodeSnippet/CodeSnippet.tsx

```diff
@@ -30,14 +30,15 @@ type Props = {
   customStyle?: any;
 };
 
-export const CodeSnippet = ({
-  text,
-  language,
-  showLineNumbers = false,
-  showCopyCodeButton = false,
-  highlightedNumbers,
-  customStyle,
-}: Props) => {
+export const CodeSnippet = (props: Props) => {
```

Comment:
I'm wondering do we need an ADR around this for any `core` packages?

Reply:
Yep I think it makes sense, but after this is through :>